apps/rotm/controllers/report.js

```diff
@@ -9,4 +11,16 @@ module.exports = class ReportController extends BaseController {
       items: req.translate('fields.description.items')
     });
   }
+
+  validateField(key, req) {
+    if (key === 'description') {
+      if (!req.form.values.url && !req.form.values[key]) {
+        return new ErrorController(key, {
+          key,
+          type: 'required'
+        });
+      }
```

Comment:
Not a blocker, but this code would probably be clearer if it just used `'description'` as a string literal instead of the variable `key`.